swarmrl/utils/utils.py

```diff
@@ -271,3 +272,36 @@ def record_training(training_dict: dict):
         training_data,
         allow_pickle=True,
     )
+
+
+def calc_signed_angle_between_directors(my_director, other_director):
+    """
+    In 2D compare two different normalized
+    directors to determine the angle between them
+
+    Parameters
+    ----------
+
+    my_director : np.ndarray
+            Normalised director in 3D.
+    other_director : np.ndarray
+            Normalised director in 3D.
+    Returns
+    singed float which represents the signed angle of my_director to otherdirector
```

Comment:
Update the doc string to:

```python
def calc_signed_angle_between_directors(
    my_director: np.ndarray, other_director: np.ndarray
) -> float:
    """
    In 2D compare two different normalized
    directors to determine the angle between them

    Parameters
    ----------
    my_director : np.ndarray
            Normalised director in 3D.
    other_director : np.ndarray
            Normalised director in 3D.
    Returns
    -------
    signed_angle : float
            Signed float which represents the signed angle of my_director to other director
    """
```
There is also a spelling mistake for singed - signed